pkgs/development/python-modules/pycnite/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "pycnite";
+  version = "2023.10.11";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
    inherit version;
    pname = "pycnite";
```

This allows to still download the source when users add something to `pname`.